docs/pages/client_usage.mdx

```diff
@@ -0,0 +1,229 @@
+
+
+## Get A Protected Object From Your Federation
+
+Protected namespaces require that a Pelican client prove it is allowed to access objects from the namespace before the object can be downloaded. In many cases, Pelican clients can guide users through the process of acquiring a token by initiating an OpenID-Connect (OIDC) flow that uses an external log-in service (such as [CILogon](https://www.cilogon.org)) through your browser. In other cases, a token must be provided to the Pelican client manually.
+
+### For Issuers That Support CILogon Code Flow
+
+Some origins support authentication with CILogon's OIDC client. In these cases, the Pelican client is capable of guiding users through authentication with the origin, once you logged in via CILogon. To download protected objects from origins that support CILogon, run the same command as for downloading a public object:
+```bash
+pelican object get pelican://<federation-url></namespace-prefix></path/to/file> <local/path/to/file>
+```
+
+If you're doing this for the very first time, Pelican will create an encrypted token wallet on your system and you will be required to provide Pelican with a password for the wallet. If this isn't your first time, you will be asked to provide your already-configured password to unlock the token wallet.
+The output looks like so:
+```bash
+The client is able to save the authorization in a local file.
+This prevents the need to reinitialize the authorization for each transfer.
+You will be asked for this password whenever a new session is started.
+Please provide a new password to encrypt the local OSDF client configuration file:
+```
+Enter in a password and press `enter` to continue and you should see a response like so:
+```bash
+To approve credentials for this operation, please navigate to the following URL and approve the request:
+
+<https://some/link/here>
+```
+Pelican will display a URL in your terminal and indicate that you should visit the URL in your browser. After copying/pasting the URL to your browser, follow all the instructions there for logging in with CILogon. Once everything is successful, you should see a page like this:
+
+<ExportedImage src={"/pelican/cilogon.png"} alt={"Screenshot of CILogon success page"} />
+
+Finally, if the login is successful, Pelican will automatically fetch the token from the CILogon service and continue with the download.
+
+### For Issuers Without CILogon Support
+
+Some origins do not support authentication though CILogon. In this case, users must supply their own JWT that's signed by the origin.
+
+> We are working hard to document how to generate a token from protected origins, before it’s ready. If you have any questions or need help getting a token from the origin, please email us at [help@pelicanplatform.com](mailto:help@pelicanplatform.com) or open a discussion on our [GitHub discussions page](https://github.com/PelicanPlatform/pelican/discussions) and we are happy to assist you.
```

Comment:
"before it's ready" - this phrase feels odd to me. Do you mean "but it's not currently ready"? I'm not quite sure what is trying to be communicated here.